docs/src/styles.css

```diff
@@ -215,12 +215,6 @@
     -moz-osx-font-smoothing: grayscale;
   }
 
-  svg {
-    /* Safari messes up geometry of many basic SVGs, such as check icons in checkboxes or chevrons on menus */
-    /* Putting them on the GPU fixes that */
-    transform: translateZ(0);
-  }
-
```

Comment:
Ahem

Another Safari bug (macOS and iOS) that compounded here was that sometimes it renders the SVG geometry wrong (!). I had been debugging it on and off for a couple days before I settled on putting SVG's on the GPU, but now it's mostly gone?!

Anyway we can't have this either way, since the icons with this style also flicker when the nav is opened.

Love Safari 🫠